config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -4506,7 +4506,9 @@ private ReactiveOAuth2UserService<OidcUserRequest, OidcUser> getOidcUserService(
 			if (bean != null) {
 				return bean;
 			}
-			return new OidcReactiveOAuth2UserService();
+			OidcReactiveOAuth2UserService reactiveOAuth2UserService = new OidcReactiveOAuth2UserService();
```

Comment:
Will you please add a test that demonstrates that when there is an `ReactiveOauth2UserService` bean, this OIDC service uses it correctly?

Reply:
Done